pandas/core/construction.py

```diff
@@ -1,16 +1,51 @@
+"""
+Constructor functions intended to be shared by pd.array, Series.__init__,
```

Comment:
hmm, I would have left this where it was I think. what is the reasoning for moving this all to construction? (the array parts I am talking)

Reply:
after trying to move this back im remembering: because `core.arrays.__init__` imports a bunch of stuff keeping it there makes import order a hassle